### DISCUSSION TITLE
use presets inside of premiere/after effects?

### DISCUSSION THREAD

**Discussion by jeffiot** [ntsc-rs/ntsc-rs#202] [Q&A]
like the title says;
I'm so stoked that NTSC-RS is available as a native plugin for after effects/premiere, but i would love to be able to use all of the presets that can be downloaded, and i havent found any discussion threads or anything in the documentation about how to do that. 
is there a way to load the preset .json files into the premiere/ae-plugin?

**Comment by notdigidani** [comment]
i dont think there is. what i do is export the footage i wanna use from ntsc-rs and import it into after effects, or, you could go one by one replicating the effect in after effects

**Reply by jeffiot** [reply]
i've been considering the second option of manually recreating each preset inside of premiere/AE, but it sounds like a lot of work that could be automated. i cant code and such though, so its unfortunately nothing i can do myself.

**Reply by notdigidani** [reply]
i think it would be a hastle to make a plugin that can read a .json file inside premier/AE, so im guessing we wont get that option soon

**Reply by jeffiot** [reply]
actually, since ae can load scripts, i believe it wouldnt be **that** hard to implement. in fact, i've managed to get a "working" version of this since i last posted by asking chatGPT to write me the script, and then re-iterating on it. i say "working" in quotation marks because while it does what it should, the effect does not look the same between the standalone and after effects, which i believe boils down to the two programs interpreting the numbers differently. gonna keep trying a bit more to make it work, but at the end of the day i think an Actual Person would have to look over the code and make adjustments for it to fully work.

**Comment by valadaptive** [comment]
I'm tracking this in https://github.com/valadaptive/ntsc-rs/issues/159. It should be supported in the next release; I just need to do a bit more testing.
